shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/antlr4/imports/Keyword.g4

```diff
@@ -78,3 +90,36 @@ NAME
 PROPERTIES
     : P R O P E R T I E S
     ;
+
+VARIABLE
+    : V A R I A B L E
+    ;
+
+HINT
+    : H I N T
+    ;
+
+SOURCE
+    : S O U R C E
+    ;
+    
+STATUS
+    : S T A T U S
+    ;
+
+SHARDING
+    : S H A R D I N G
+    ;
+
+READWRITE_SPLITTING
+    : R E A D W R I T E UL_ S P L I T T I N G
+    ;
+    
+DATABASE_VALUE
+    : D A T A B A S E UL_ V A L U E
+    ;
+    
+TABLE_VALUE
+    : T A B L E UL_ V A L U E
+    ;
+    
```

Comment:
The last line is expected to be a new line instead of a line with indent.

Reply:
Sure, I'll fix it.